### PR TITLE
Adhoc config updates

### PR DESCRIPTION
**Pull request by philbogle**
Modify the adhoc config to be more like production in terms of digest and caching behavior, since this is something we need to test.

### CODE REVIEW

File: dashboard/config/environments/adhoc.rb [hunk 1]
```diff
@@ -6,4 +6,27 @@
 Dashboard::Application.configure do
   # Show full error reports
   config.consider_all_requests_local       = true
+
+  # Use digests.
```
Comment (islemaster) [inline dashboard/config/environments/adhoc.rb:10]: Might want to include this directive from the production environment too - it introduced some differences in local/production for me in recent work.

``` ruby
  # Do not fallback to assets pipeline if a precompiled asset is missed.
  config.assets.compile = false
```

File: dashboard/config/environments/adhoc.rb [hunk 2]
```diff
@@ -6,4 +6,27 @@
...
+
+  config.action_controller.perform_caching = true
+  config.serve_static_files = true
+  config.static_cache_control = "public, max-age=86400"
+
+  # Do not fallback to assets pipeline if a precompiled asset is missed.
+  config.assets.compile = false
+
+  # Version of your assets, change this if you want to expire all your assets.
+  config.assets.version = '1.0'
```
Comment (islemaster) [inline dashboard/config/environments/adhoc.rb:21]: have we ever changed this on production?

**Comment by islemaster** [comment]
LGTM!

File: dashboard/config/environments/adhoc.rb (hunk 2 above)
Comment (philbogle) [inline dashboard/config/environments/adhoc.rb:21]: None, it has never been changed.  If we do change we probably want to refactor so there is a single source of truth.

File: dashboard/config/environments/adhoc.rb (hunk 1 above)
Comment (philbogle) [inline dashboard/config/environments/adhoc.rb:10]: That is already present on line 18.

**Comment by philbogle** [comment]
Thanks for the review!

File: dashboard/config/environments/adhoc.rb (hunk 1 above)
Comment (islemaster) [inline dashboard/config/environments/adhoc.rb:10]: So it is!  My bad.

**Pull request merged**